src/js/jsx/sections/style/Type.jsx

```diff
@@ -470,7 +470,11 @@ define(function (require, exports, module) {
                 if (postScriptFamilyName) {
                     familyName = this._getPostScriptFontFamily(postScriptFamilyName);
                     if (!familyName) {
-                        placeholderText = "[" + postScriptFamilyName + "]";
+                        if (layers.size > 1) {
+                            placeholderText = strings.STYLE.TYPE.MIXED;
```

Comment:
your comment said you it would show `[Missing Font]` but this is the string for "Mixed Fonts" . I just want to double check which one we want.